client/state/initial-state.js

```diff
@@ -48,7 +48,24 @@ function deserialize( state ) {
 	return reducer( state, { type: DESERIALIZE } );
 }
 
-function loadInitialState( initialState ) {
+function addSympathy( initialStateLoader ) {
+	if ( 'development' !== process.env.NODE_ENV || ( Math.random() > 0.5 ) ) {
+		return initialStateLoader;
+	}
+
+	console.log( 'Skipping initial state load to recreate first-load experience.' ); // eslint-disable-line no-console
+
+	try {
+		localStorage.clear();
+		indexedDB.deleteDatabase( 'calypso' );
```

Comment:
We should be using the interface here. localforage may use other drivers, so this will clear state on say chrome, but not in another browser.